src/VisualStudio/Core/Def/Implementation/Venus/ContainedDocument.cs

```diff
@@ -805,6 +823,10 @@ private void AdjustIndentation(IProjectionBuffer subjectBuffer, IEnumerable<int>
             Contract.Requires(object.ReferenceEquals(originalText, snapshot.AsText()));
 
             var root = document.GetSyntaxRootAsync(CancellationToken.None).WaitAndGetResult(CancellationToken.None);
+            if (root == null)
```

Comment:
I think it woudl be better to change this to:

```c#
if (!document.SupportsSyntax)
{
    return;
}
```

however, in the future, we're going to want to use the IEditorFormattingService (not quite sure the exact name) so we get appropriate TypeScript indentation here as well.